i18n/.js/src/main/scala/io/udash/i18n/bindings/TranslationBindingHelper.scala

```diff
@@ -0,0 +1,48 @@
+package io.udash.i18n.bindings
+
+import io.udash.logging.CrossLogging
+import org.scalajs.dom._
+import org.scalajs.dom.raw.Node
+
+private[bindings] trait TranslationBindingHelper extends CrossLogging {
+  import TranslationBindingHelper._
+
+  implicit def String2Node(str: String): Node =
+    document.createTextNode(str)
+
+  implicit def String2Nodes(str: String): Seq[Node] =
+    try {
+      val wrapper = document.createElement("div")
+      wrapper.innerHTML = str
+      val kids = wrapper.childNodes
+      kids.length match {
+        case l if l > 0 =>
+          (0 until kids.length).map(kids(_))
+        case _ =>
+          logger.warn(s"'$str' is parsed as nothing, use it as string.")
+          val node: Node = document.createTextNode(str)
+          Seq(node)
+      }
+    } catch {
+      case e: Throwable =>
```

Comment:
Can we narrow this to a more specific class of exceptions? This may very well have been an exception from `document.createTextNode(str)` and fail again.